common/persistence/nosql/nosqlplugin/cassandra/session.go

```diff
@@ -23,18 +23,22 @@ package cassandra
 import (
 	"time"
 
+	"github.com/uber/cadence/environment"
+
 	"github.com/uber/cadence/common/config"
 	"github.com/uber/cadence/common/persistence/nosql/nosqlplugin/cassandra/gocql"
 )
 
 const (
-	cassandraProtoVersion = 4
 	defaultSessionTimeout = 10 * time.Second
 )
 
 // CreateSession creates a new session
 // TODO this will be converted to private later, after all cassandra code moved to plugin pkg
 func CreateSession(cfg config.Cassandra) (gocql.Session, error) {
+	if cfg.ProtoVersion == 0 {
+		cfg.ProtoVersion = environment.CassandraDefaultProtoVersionInteger
+	}
```

Comment:
I believe this is not needed as the default will be set within `CreateSession`, can you double check?

Reply:
You're right, removed